src/app/wizard-extension/organisation-wizard-extension/organisation-wizard-extension.component.html

```diff
@@ -256,42 +248,34 @@
-                                </div>
                             </div>
                         </mat-panel-title>
                     </mat-expansion-panel-header>
 
-                    <div class="mb-3">
+                    <div hidden class="mb-3">
                         <label class="form-label">Private Key</label>
-                        <textarea rows="10" class="form-control" [(ngModel)]="orgaMetadata.organisationSignerConfigDto.privateKey">
+                        <textarea rows="10" class="form-control" [value]="orgaMetadata.organisationSignerConfigDto.privateKey" [disabled]="true">
                         </textarea>
                         <div cFormText style="color: #7f7f7f;">
-                            <small>Der Private Key zur unten genannten Verifizierungsmethode. Wird zum Signieren von Selbstbeschreibungen verwendet.</small>
-                        </div>
-                        <div *ngIf="!isFieldFilled(orgaMetadata.organisationSignerConfigDto.privateKey)" style="color: red">
-                            <small>
-                                Private Key ist erforderlich.
-                            </small>
+                            <small>Der Private Key zur unten genannten Verifizierungsmethode. Kann zum Signieren von Selbstbeschreibungen verwendet werden.</small>
                         </div>
                     </div>
 
-                    <div class="mb-3">
+                    <div hidden class="mb-3">
                         <label class="form-label">Verifizierungsmethode</label>
-                        <input class="form-control" [(ngModel)]="orgaMetadata.organisationSignerConfigDto.verificationMethod">
+                        <input class="form-control" [value]="orgaMetadata.organisationSignerConfigDto.verificationMethod" [disabled]="true">
                         <div cFormText style="color: #7f7f7f;">
                             <small>Die zugehörige Verifizierungsmethode zum oben genannten Private Key, welche den Public Key referenziert. Derzeit wird ausschließlich did:web unterstützt.</small>
                         </div>
-                        <div *ngIf="!isFieldFilled(orgaMetadata.organisationSignerConfigDto.verificationMethod)" style="color: red">
-                            <small>
-                                Verifizierungsmethode ist erforderlich.
-                            </small>
+                    </div>
+
+                    <div class="mb-3">
+                        <label class="form-label">Merlot-Verifizierungsmethode</label>
+                        <input class="form-control" [value]="orgaMetadata.organisationSignerConfigDto.merlotVerificationMethod" [disabled]="true">
+                        <div cFormText style="color: #7f7f7f;">
+                            <small>Die Merlot-Verifizierungsmethode (did:web), welche zur Zeit standardmäßig von allen Mitgliedern zum Signieren von Selbstbeschreibungen genutzt wird.</small>
```

Comment:
this is not quite right, the merlot verification method is specific to a particular participant, though the referenced key is used by all

Reply:
i hope the new description fits better